### ISSUE TITLE
Increase text contrast in navbar and footer

### ISSUE THREAD

**Issue by allie-tatarian** [katieefrey/metasat#10]
Non-active links in the navbar have a contrast ratio of 4.2, text in the footer has a contrast ratio of 4.17; the ideal is 4.5 or higher.

Either the navbar can be colored darker, or the non-active link text can be made lighter. The footer can be made lighter, or the text color changed to white or light grey.

**Comment by katieefrey** [comment]
For the navbar, this site is using the default "navbar-dark" setting that Bootstrap has.  It automatically defines the color of the background, the text and the active/inactive links.

I found this statement from Bootstrap:

Color contrast

Most colors that currently make up Bootstrap’s default palette—used throughout the framework for things such as button variations, alert variations, form validation indicators—lead to insufficient color contrast (below the recommended WCAG 2.0 color contrast ratio of 4.5:1) when used against a light background. Authors will need to manually modify/extend these default colors to ensure adequate color contrast ratios.
https://getbootstrap.com/docs/4.0/getting-started/accessibility/#color-contrast

Looks like this is still an open issue at least...
https://github.com/twbs/bootstrap/issues/25126

Ah, and a draft pull request that addresses contrast issues from April 2020, last updated about a month ago:
https://github.com/twbs/bootstrap/pull/30548

Rather than editing the navbar colors myself, what do you think about throwing our support in for this pull request?



The footer, however, is entirely my own color creation, so I'll look at that!  (tho now I am wondering if bootstrap has default footer colors options hmm.)

**Comment by katieefrey** [comment]
I've been mildly unhappy with the footer for a while.  So I reworked it.  The changes aren't live on the website yet, but I pushed the footer changes to the repo.

Except for the index/home page, the footer now has a white background and blank text, like the rest of the page.  Also, except for the index/home page, the footer is now a file include.  I had to make some minor edits to every html template (including your resources, basic, advanced, linked data, etc pgs).  Basically, I wrapped all the page content into another div called... "mainstuff" because I am the best at naming things, and set that div to have a min height of 100% of the window (minus some amount for the navbar).

**Comment by katieefrey** [comment]
ok, the changes are now live.  I also normalized the height of the navbar across desktop and mobile by giving the logo image a static height.  Now I am able to give a static height to the page content that uses the maximum amount of page real estate.  This will be useful when working further on the mobile version.

**Comment by katieefrey** [comment]
nav bar color has been darkened, contrast is now 4.6.

working on putting a transparency behind the front page text to increase the contrast

**Comment by katieefrey** [comment]
Updated the main home page.  Had to do a fair amount of html and css clean up and editing.  I think the main page looks better on mobile as well.

**Comment by katieefrey** [comment]
contrast fixes have survived the mobile update, and i've also fixed the contrast of the "find" button by changing its background color.
